src/server/pfs/fuse/mounter.go

```diff
@@ -2,13 +2,13 @@ package fuse
 
 import (
 	"fmt"
-	"log"
 	"os"
 	"os/signal"
 	"sync"
 
 	"bazil.org/fuse"
 	"bazil.org/fuse/fs"
+	log "github.com/Sirupsen/logrus"
```

Comment:
We should either be using it everywhere or using it nowhere. I'm only using it here because it's what's used in the rest of the fuse package so I need to pass the `SetLevel` call to it for it to do anything.